mytoyota/utils/models.py

```diff
@@ -0,0 +1,25 @@
+"""Utilities for manipulating or extend pydantic models."""
+from pydantic import BaseModel, root_validator
+
+
+class CustomBaseModel(BaseModel):
+    """Model that automatically returns values that cannot be parsed correctly as 'None' value."""
+
+    @root_validator(pre=True)
+    def invalid_to_none(cls, values: dict[str, object]) -> dict[str, object]:  # noqa: N805
```

Comment:
I dont quite understand this function. You pass it values which appears to be used to create a validatde_values dictionary but then return what you were passed that may have been updated if an error was found. Either validated_values is not needed or it was meant to be returned?

Reply:
For explanation please have a look at: https://stackoverflow.com/a/76674285

Reply:
Can you please have another Look at this @GitOldGrumpy